src/EnergyPlus/PlantChillers.cc

```diff
@@ -6503,6 +6551,17 @@ namespace PlantChillers {
                 }
             }
 
+            if (!state.dataIPShortCut->lAlphaFieldBlanks(9)) {
+                thisChiller.thermosiphonTempCurveIndex = Curve::GetCurveIndex(state, Util::makeUPPER(state.dataIPShortCut->cAlphaArgs(9)));
+                if (thisChiller.thermosiphonTempCurveIndex == 0) {
+                    ShowSevereError(state, format("{}{}=\"{}\"", RoutineName, state.dataIPShortCut->cCurrentModuleObject, thisChiller.Name));
+                    ShowContinueError(state,
+                                      format("Invalid {} = {}", state.dataIPShortCut->cAlphaFieldNames(9), state.dataIPShortCut->cAlphaArgs(9)));
+                    ErrorsFound = true;
+                }
```

Comment:
Test using FreeCoolingChiller_wWaterSideEconomizer:

    ** Severe  ** GetConstCOPChillerInput: Chiller:ConstantCOP="MAIN CHILLER"
    **   ~~~   ** Invalid Thermosiphon Capacity Fraction Curve Name = THERMOCAPFRACCURVE2
    **  Fatal  ** Errors found in processing input for Chiller:ConstantCOP